tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -2,6 +2,14 @@
 
 from coalib.bearlib.aspects.collections import aspectlist
 from coalib.bearlib.aspects.Metadata import Metadata
+from coalib.bears.LocalBear import LocalBear
+
+
+class aspectsTestBear(LocalBear, aspects={
+        'detect': [Metadata.CommitMessage.Shortlog],
+        'fix': [Metadata.CommitMessage.Shortlog.TrailingPeriod],
+}):
+    LANGUAGES = {'Python'}
```

Comment:
Like mentioned in `aspectlist.__contains__`, better define a new meta keyword-only `languages=` attribute for `class` definition in `bearclass`